src/pages/components/number-input/style.mdx

```diff
@@ -31,16 +29,16 @@ input color is `$field-02` and is used on `$ui-01` page backgrounds.
 
 ### Interactive states
 
-| Class                                       | Property         | Color token    |
-| ------------------------------------------- | ---------------- | -------------- |
-| `.bx--number:focus`                         | border           | `$focus`       |
-| `.bx--number__controls:focus`               | border           | `$focus`       |
-| `[data-invalid]`                            | border           | `$support-01`  |
-| `[data-invalid]:focus`                      | color            | `$support-01`  |
-| `.bx--form-requirement`                     | text color       | `$support-01`  |
-| `.bx--label:disabled`                       | text color       | `$disabled-02` |
-| `.bx--number:disabled`                      | background-color | `$disabled-01` |
-| `.bx--number input[type='number']:disabled` | text color       | `$disabled-03` |
+| Class                                       | Property         | Color token               |
+| ------------------------------------------- | ---------------- | ------------------------- |
+| `.bx--number:focus`                         | border           | `$focus`                  |
+| `.bx--number__controls:focus`               | border           | `$focus`                  |
+| `[data-invalid]`                            | border           | `$support-error`          |
+| `[data-invalid]:focus`                      | color            | `$support-error`          |
+| `.bx--form-requirement`                     | text color       | `$support-error`          |
+| `.bx--label:disabled`                       | text color       | `$text-disabled`          |
+| `.bx--number:disabled`                      | background-color | `$layer-disabled`         |
+| `.bx--number input[type='number']:disabled` | text color       | `$text-on-color-disabled` |
```

Comment:
I think was a wrong token before should have been `$disabled-02` here so now its just `$text-disabled`